ext/pdo_pgsql/tests/bug_last_insert_id.phpt

```diff
@@ -0,0 +1,42 @@
+--TEST--
+currval() vs lastval()
+PDO PgSQL Bug #1134 [BUG] New record, PostgreSQL and the Primary key https://github.com/phalcon/cphalcon/issues/1134
+Fix PHP original PDO_PGSQL::lastInsertId https://github.com/doctrine/dbal/pull/2329
```

Comment:
This section should contain just one line, usually. As when multiple tests run, everyone shows up as one line in the out. The other info can be added as a comment to the script in the --FILE-- section.

Thanks.